showcase/README.md

```diff
@@ -58,93 +66,132 @@ $ gateway --help
+* retaining access to your cryptid account via the additional key
+
+You can go through the steps below and follow along on YouTube: https://www.youtube.com/watch?v=72Oo11qy7ug
 
 ```shell
+# Initialize cryptid configuration (using your solana key by default)
 $ cryptid init
 
+# View the cryptid configuration
 $ cryptid config
 
-$ cryptid document (shows same as above)
+# View your DID document
+$ cryptid document
 
+# Airdop SOL to the signing key and cryptid account
 $ cryptid airdrop
 
+# Generate a new Solana key
 $ solana-keygen new -o key2.json
 
+# Add the new public key to your cryptid account, with 'key2' as an alias
 $ cryptid key add <pubkey> key2
 
+# Transfer SOL to the new key
 $ cryptid transfer <pubkey> 1000000
 
+# View the updated DID document (with the additional key added)
 $ cryptid document
 
+# Update cryptid configuration to use the new key by default
 $ cryptid config set keyFile $(pwd)/key2.json
 
+# Remove the default key
 $ cryptid key remove default
 
+# Check the updated DID document
 $ cryptid document
 
-# Show that the default key is no longer able to transfer
+# Update cryptid configuration to use the original (default) key
 $ cryptid config set keyFile $HOME/.config/solana/id.json
 
+# Expect transfer of using the removed key to fail
 $ cryptid transfer <pubkey> 1000000
 
+# Switch back to your new key
 $ cryptid config set keyFile $(pwd)/key2.json
 ```
 
 ## Controllers
 
+Here we show how you can use [Cryptid](https://github.com/identity-com/cryptid) to setup one DID to be a controller of 
+another DID. For example, if a DID represents a business in the real world, the CEO's DID can be setup as a 
+controller of the business. This will allow the CEO to sign as that business using their own keys.
 
-Goal: Show the power of DID Controllers for joining accounts
+You can go through the steps below and follow along on YouTube: https://www.youtube.com/watch?v=4oSTRnvnmNM
 
 ```shell
+# Create a new key to demonstrate the controlled cryptid account
 $ solana-keygen new -o controlled.json
 
+# Create a cryptid configuration for the new key
 $ cryptid init -k controlled.json -p controlled.yml
 
+# View the configuration
 $ cryptid config -c controlled.yml
 
-# Fund new Cryptid address
+# Fund new cryptid address
 $ cryptid transfer <NEW_DID> 100000000
 
 # Fund Signer #default signer
 $ cryptid transfer <pubkey controlled.json> 100000000
 
+# Add your main account DID 
 $ cryptid controller add -c controlled.yml <OLD_DID>
 
+# Add an alias for the controlled did (for convenience)
 $ cryptid alias controlled <NEW_DID>
 
+# Add an alias for your original did (for convenience)
 $ cryptid alias me <OLD_DID>
 
+# Check the DID document for the controlled cryptid
 $ cryptid document --as controlled
 
+# Check the balance for the controlled cryptid
 $ cryptid balance --as controlled
 
+# Transfer SOL on behalf of the controlled cryptid account using the original cryptid account
```

Comment:
"On behalf of" is a little confusing here because the funds are leaving the controlled account
"using the original cryptid account" is also a little hard to understand- what does "using " mean exactly?